docs/sources/whatsnew/whats-new-in-v11-0.md

```diff
@@ -0,0 +1,381 @@
+---
+description: Feature and improvement highlights for Grafana v11.0-preview
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '11.0'
+  - '11.0-preview'
+  - release notes
+labels:
+products:
+  - cloud
+  - enterprise
+  - oss
+title: What's new in Grafana v11.0-preview
+weight: -42
+---
+
+<!-- vale GoogleWe = NO -->
+<!-- vale We = NO -->
+
+# What’s new in Grafana v11.0-preview
+
+Welcome to Grafana 11.0-preview! This preview release contains some notable improvements, most notably, the ability to explore your metrics without queries. We've taken strides to improve the dashboard experience with subfolders becoming generally available and the addition of enhanced flowcharting capabilities. We've also migrated the dashboard viewing experience so that it's using our Scenes library.
```

Comment:
```suggestion
Welcome to Grafana 11.0-preview! This preview release contains several improvements, most notably, the ability to explore your metrics without queries. We've taken strides to improve the dashboard experience with subfolders becoming generally available and the addition of enhanced flowcharting capabilities. We've also migrated the dashboard viewing experience so that it's using our Scenes library.
```